.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+
+      - name: Install purescript-psa
+        run: npm install -g purescript-psa
+
+      - uses: thomashoneyman/setup-purescript@v1
+        with:
+          purescript: "0.13.8"
+
+      - name: Cache PureScript dependencies
+        uses: actions/cache@v2
+        with:
+          key: ${{ runner.os }}-spago-${{ hashFiles('**/*.dhall') }}
+          path: |
+            .spago
+            output
```

Comment:
Caching the `output` directory means that the compiler will only rebuild files that have changed since the last successful cache. I've tested this in some other projects (halogen-hooks, etc.) and it seems pretty reliable -- files are rebuilt when they've changed since the last run in CI. However, we could also just cache the `.spago` directory and always compile from scratch.